website/docs/cloud-docs/vcs/bitbucket-data-center.mdx

```diff
@@ -97,7 +97,7 @@ Do not specify a passphrase because Terraform cannot use SSH keys that require a
 In the following steps, you must provide HCP Terraform with the private SSH key you created in [Create an SSH key for Terraform](#create-an-ssh-key-for-terraform). Although HCP Terraform does not display the text of the key to users after it is entered, it retains the key and uses it for authenticating to Bitbucket Data Center.
 
 1. If you are logged into Bitbucket Data Center as an administrator, log out before proceeding.
-1. Log in with the account that you want to enable HCP Terraform or Terraform Enterprise to log in with. Many organizations use a dedicated service user account for this purpose. The account you use for connecting HCP Terraform must have admin access to any shared repositories of Terraform configurations because since creating webhooks requires admin permissions. Refer to [Requirements](#requirements) for additional information.
+1. Log in with the account that you want to HCP Terraform or Terraform Enterprise to log in with. Many organizations use a dedicated service user account for this purpose. Because creating webhooks requires admin permissions, the account must have admin access to any shared repositories of Terraform configurations. Refer to [Requirements](#requirements) for more information.
```

Comment:
```suggestion
1. Log in with the account that you want to HCP Terraform or Terraform Enterprise. Many organizations use a dedicated service user account for this purpose. Because creating webhooks requires admin permissions, the account must have admin access to any shared repositories of Terraform configurations. Refer to [Requirements](#requirements) for more information.
```

Reply:
@im2nguyen This suggestion doesn't make sense.